src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,9 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from "../CommentInfo/CommentInfo";
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
```

Comment:
There is no error handling for the scenario where the 'comments' array might be null or undefined. It's recommended to add error handling or default props.